git-cmake-format.py

```diff
@@ -24,7 +24,7 @@ def getGitHead():
 def getGitRoot():
     RevParse = subprocess.Popen([Git, 'rev-parse', '--show-toplevel'],
         stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    return RevParse.stdout.read().strip()
+    return RevParse.stdout.read().decode().strip()
```

Comment:
Does this also work for Python 2?

Reply:
To be honest this implementation should probably just be replaced with a call to `subprocess.check_output`.

Reply:
> Does this also work for Python 2?

Works for me with both Python 2.7 and Python 3.6